specification/operationalinsights/resource-manager/readme.md

```diff
@@ -62,6 +62,22 @@ input-file:
 - Microsoft.OperationalInsights/preview/2019-08-01-preview/OperationalInsights.json
 ```
 
+``` yaml $(tag) == 'package-2020-03-preview'
```

Comment:
Could you change the default tag to this one too?

Reply:
I changed it. Please check it.

Reply:
good